src/bundle/Resources/public/js/scripts/core/collapse.js

```diff
@@ -1,11 +1,58 @@
-(function (global, doc) {
+(function (global, doc, bootstrap) {
+    let toggleAllTimeout;
+    const toggleAllBtn = doc.querySelector('.multi-collapse-btn');
+    const expandAll = toggleAllBtn?.querySelector('.ibexa-attribute-group__toggler-expand');
+    const collapseAll = toggleAllBtn?.querySelector('.ibexa-attribute-group__toggler-collapse');
+    const MULTI_COLLAPSE_BODY_CLASS = '.multi-collapse';
+    let singleElementClicked = [];
+    const checkIfChangeText = () => {
```

Comment:
I think the name of the function is wrong because this function only toggles the d-none class

Reply:
I proposed a new name for the function. In my opinion, based on the YAGNI principle, we should first discuss the probability of such a situation before making such changes.